specs/scoring_spec.rb

```diff
@@ -42,21 +43,33 @@
 
   describe 'highest_score_from' do
     it 'returns nil if no words were passed' do
+      Scrabble::Scoring.highest_score_from([]).must_be_nil
     end
 
-    it 'returns the only word in a length-1 array' do
+    it 'returns the only word in a length 1 array' do
+      words1 = ["cookie"]
+      Scrabble::Scoring.highest_score_from(words1).must_equal "cookie"
     end
 
-    it 'returns the highest word if there are two words' do
+    it 'returns the highest value word if there are two words' do
+      words2 = ["cookie", "cake"]
+      Scrabble::Scoring.highest_score_from(words2).must_equal "cookie"
     end
```

Comment:
Like we discussed in class, it would probably be wise to check multiple orderings of these words, that is, both `['dog', 'fish']` and `['fish', 'dog']`. You'd be surprised how often this sort of thing turns up a bug.